### PR TITLE
Allow the versions to be passed in as an argument to the installer

### LINKED ISSUE #1927: Install script should be generalized so it will install any version and does not reference a specific version
Install script should be generalized so it will install any version and does not reference a specific version

### PR DESCRIPTION
**Pull request by kholdaway**
Closes #1927

### CODE REVIEW

**Comment by kholdaway** [comment]
This script was able to install 0.0.45 and then upgrade to 0.0.46.  In other words, the install script is not tied to a specified version.

File: install/install.sh [hunk 1]
```diff
@@ -39,6 +33,10 @@ usage() {
 
 
     Extra Variables:
+    * Specify the version of the quipucords server to install.   (defaults to the latest release):
```
Comment (abaiken) [inline install/install.sh]: Remove the awkward spacing before `(defaults...`

File: install/install.sh [hunk 2]
```diff
@@ -39,6 +33,10 @@ usage() {
...
+         -e server_version=0.0.46
+    * Specify the version of the qpc cli to install.   (defaults to the latest release):
```
Comment (abaiken) [inline install/install.sh]: Same comment as above

File: install/roles/qpc_server/tasks/main.yml [hunk 3]
```diff
@@ -31,41 +31,89 @@
...
 
-- name: Set server image name
+- name: Set server docker image name
   set_fact:
-    image: "{{ ansible_env['SERVER_IMAGE'] | default('quipucords') }}"
-  when: image is not defined
+    server_docker_image_name: "quipucords"
 
-- name: Set server_image_and_tag
+- name: Set Quipucords docker image and tag
```
Comment (abaiken) [inline install/roles/qpc_server/tasks/main.yml]: not a blocker at all but there are a lot of inconsistencies within the name descriptions, for example, Quipucords vs quipucords, Postgres vs postgres

File: install/install.sh (hunk 1 above)
Comment (kholdaway) [reply install/install.sh]: Done

File: install/install.sh (hunk 2 above)
Comment (kholdaway) [reply install/install.sh]: Done

File: install/roles/qpc_server/tasks/main.yml (hunk 3 above)
Comment (kholdaway) [reply install/roles/qpc_server/tasks/main.yml]: Fixed

File: install/roles/prep/tasks/main.yml [hunk 4]
```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Checking if server_version is defined when install_offline is true
+  fail:
+  when:
+    - install_offline is defined and server_version is not defined
```
Comment (abaiken) [inline install/roles/prep/tasks/main.yml]: What happens if `install_offline` is defined but is set to false?

Comment (kholdaway) [reply install/roles/prep/tasks/main.yml]: Fixed.

**Pull request merged**